README.md

```diff
@@ -80,6 +80,25 @@ Because the browser is controlled by a script the actions might be too fast to
 watch. Use the `AGAMA_SLOWMO` variable with a delay in miliseconds between the
 actions. A reasonable value is round 50.
 
+### Perform installation
+
+The test has te ability to install OpenSUSE Tumbleweed, following the default
```

Comment:
only mention the environment variable and its meaning, agama can install multiple products.

Reply:
Solved here bbf69efe39e06cc95d295f663d3b220933fc32d0